Rewrite intro text on User manual's Command monitoring index page (#4088)
wazuh/wazuh-documentation: source/user-manual/capabilities/command-monitoring/index.rst

Command monitoring
==================

> Wazuh not only can watch log files or Windows events on its monitored hosts for applications and operating system log messages, or watch for messages of devices that are using Syslog, but it can also read the output of monitored commands to be run periodically on these hosts. These specifically authorized commands can be configured to check important system status data such as running processes, system load averages, or to detect USB storage, among other monitoring purposes.

Reviewer: ```suggestion
Wazuh strengthens your system monitoring capabilities by allowing you to read the output of monitored commands to be run periodically on hosts. These specifically authorized commands can be configured to detect USB storage or check important system status data, such as running processes and system load averages, among other monitoring purposes.
```
